tests/lightning_mode/conf/dm-task.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: test
+task-mode: all
+is-sharding: false
+meta-schema: "dm_meta"
+# enable-heartbeat: true
+heartbeat-update-interval: 1
+heartbeat-report-interval: 1
+
+target-database:
+  host: "127.0.0.1"
+  port: 4000
+  user: "root"
+  password: ""
+  session:
+    tidb_skip_utf8_check: 1
+    tidb_disable_txn_auto_retry: off
+    tidb_retry_limit: "10"
+
+mysql-instances:
+  - source-id: "mysql-replica-01"
+    black-white-list:  "instance" # compatible with deprecated config
+    mydumper-config-name: "global"
+    loader-config-name: "global"
+    syncer-config-name: "global"
+    filter-rules: ["user-filter-1"]
+
+  - source-id: "mysql-replica-02"
+    block-allow-list:  "instance"
+    mydumper-config-name: "global"
+    loader-config-name: "global"
+    syncer-config-name: "global"
+    filter-rules: ["user-filter-1"]
+
+black-white-list: # compatible with deprecated config
+  instance:
+    do-dbs: ["all_mode"]
+    ignore-dbs: ["ignore_db"]
+
+filters:                     # filter rules, mysql instance can ref rules in it
+  user-filter-1:
+    schema-pattern: "*"    # pattern of the upstream schema name, wildcard characters (*?) are supported
+    table-pattern: "*"        # pattern of the upstream table name, wildcard characters (*?) are supported
+    events: ["truncate table", "drop table"]  # ignore truncate/drop table ddl
+    action: Ignore
+
+mydumpers:
+  global:
+    threads: 4
+    chunk-filesize: 64
+    skip-tz-utc: true
+    extra-args: ""
+
+loaders:
+  global:
```

Comment:
didn't set lightning backend. and I think we can use one of below ways to test

- add a new case of `lightning_mode`, the database name of this test should be `lightning_mode` not `all_mode`, and `lightning_mode` should be added into `other_integration_1.txt` (or 2)
- just change an existing case (like all_mode), modify the task configuration to switch to lightning loader